GoMeet/src/main/webapp/js/meeting-location-dao.js

```diff
@@ -1,26 +1,128 @@
+/** Constructs a Data Access Object for location data. */
+class MeetingLocationDaoFactory {
+  /** 
+   * Returns a meetingLocationDAO object based on whether the map stores to a temporary
+   * or permanent location.
+   */
+  static getLocationDao(storageType) {
+    if (storageType === 'permanent') {
+      return new PermMeetingLocationDao();
+    } else {
+      return new TempMeetingLocationDao();
+    }
+  }
+}
+
+/** Stores location data to session storage. */
```

Comment:
Might be good to mention (either in the comments or in your design docs) why you can't use the saveMeeting() function to save the location data to session storage.

Reply:
Yup, will do :)